torch/backends/cuda/__init__.py

```diff
@@ -27,10 +27,12 @@
 
 
 def is_built():
-    r"""Returns whether PyTorch is built with CUDA support.  Note that this
-    doesn't necessarily mean CUDA is available; just that if this PyTorch
-    binary were run a machine with working CUDA drivers and devices, we
-    would be able to use it."""
+    r"""
+    Return whether PyTorch is built with CUDA support.
+
+    Note that this doesn't necessarily mean CUDA is available; just that if this PyTorch
+    binary were run a machine with working CUDA drivers and devices, we would be able to use it.
```

Comment:
Pre-extincting, but please fix
```suggestion
    binary were run on a machine with working CUDA drivers and devices, we would be able to use it.
```